src/Package/Impl/DataInspect/DataSource/GridDataSource.cs

```diff
@@ -31,5 +30,10 @@ internal sealed class GridDataSource {
                 }
             }
         }
+
+        public static Task<IGridData<string>> GetGridDataAsync(string expression, GridRange? gridRange, ISortOrder sortOrder = null) =>
+            GetGridDataAsync(
+                VsAppShell.Current.ExportProvider.GetExportedValue<IRSessionProvider>().GetInteractiveWindowRSession(),
```

Comment:
ok

---
In reply to: [67211001](https://github.com/Microsoft/RTVS/pull/1927#discussion_r67211001) [](ancestors = 67211001)